src/current/_includes/sidebar-data-v23.1.json

```diff
@@ -12,9 +12,6 @@
   {% include_cached v23.1/sidebar-data/migrate.json %},
   {% include_cached v23.1/sidebar-data/schema-design.json %},
   {% include_cached v23.1/sidebar-data/reads-and-writes.json %},
-  {% include_cached v23.1/sidebar-data/develop.json %},
```

Comment:
I don't see any of the `cockroachcloud` pages you modified in this PR expand in the sidenav, and I suspect it's because you need the following somewhere here or in a different file:

```
{% include_cached v23.1/sidebar-data/cloud-deployments.json %}
{% include_cached v23.1/sidebar-data/self-hosted-deployments.json %}
```

Reply:
Duh 🤦